python/paddle/distributed/auto_parallel/process_mesh.py

```diff
@@ -204,7 +205,32 @@ def __getitem__(self, index):
             else:
                 return ProcessMesh([new_mesh])
 
-    def get_dim_size(self, dim_name):
+    def get_rank_by_dim(self, dim: Union[str, int], process_id: int) -> int:
```

Comment:
the function is actually `get_rank_by_dim_and_process_id`

Reply:
Done